functionary/prompt_template/llama3_prompt_template.py

```diff
@@ -28,6 +28,7 @@ class Llama3Template(PromptTemplate):
     function_separator = "<|reserved_special_token_249|>"
     version = "v2.llama3"
     fn_param_sep_token = "\n"
+    tokenizer_name_or_path = "meetkai/functionary-small-v2.5"
```

Comment:
Actually, should we add this? because the prompt templates can be used with any pretrained models (like v2.5) or exclusively for a model such as v2.llama3

Reply:
In addition, when we `**train a new model**`, we don't have the model ahead. For example, when I implement a prompt template for Phi-3, I don't have the model ahead, so I don't know what to set  for the value of tokenizer_name_or_path

Reply:
I added this in order to load the tokenizer from_pretrained to test the request handling for streaming. We discussed previously and concluded that it is weird to store the files of the tokenizer in the repo itself.

Reply:
I think we should decouple it because it is only used for testing (most of people don't care about testing, only us when we implement a new template) , we can create a map from prompt_template --> pretrained_path in tests/test_request_handling.py